packages/kit/src/utils/http.js

```diff
@@ -19,3 +19,55 @@ export function to_headers(object) {
 
 	return headers;
 }
+
+/**
+ * @param {string} accept
```

Comment:
might not hurt to add descriptions for these two. I guess it's the accept header and types that we're okay using or something?

Reply:
added